src/google/protobuf/compiler/cpp/cpp_helpers.cc

```diff
@@ -1130,7 +1130,7 @@ bool IsImplicitWeakField(const FieldDescriptor* field, const Options& options,
 
 MessageAnalysis MessageSCCAnalyzer::GetSCCAnalysis(const SCC* scc) {
   if (analysis_cache_.count(scc)) return analysis_cache_[scc];
-  MessageAnalysis result{};
+  MessageAnalysis result = {};
```

Comment:
This one should be unneeded

Reply:
You're right - not sure what I was seeing, but it doesn't seem to be needed.